DATA_FORMAT.md

```diff
@@ -0,0 +1,83 @@
+# JSON Data fields
+
+## Message Data
+These fields are the primary data fields containing the most basic message data and used to identify the specific device. 
+For some devices these are the *only* fields contained in the message, as the message itself consitutes an event from this
+particular device model.
+
+* **time** (string) (Required)
+  * Time stamp. String containing date and time of when the message was received. Format is dependent on
+    current locale unless the `-U` command line argument is used.
+
+* **model** (string) (Required)
+  * Device model. Human readable string concisely describing the device by manufacturer name 
+    and manufacturers model designation according to the following syntax: `"<Manufacturer>, <Model>"`. 
```

Comment:
Can you change this to read "<Manufacturer>-<Model>"?

Reply:
Regarding units, I suggest changing from using 'p' as division ('per') to underscore '_':
Examples:
m_s (meters per second)
mi_h (miles per hour)
in_h (inches per hour)
The reason is to avoid ambiguity like the common "mph".

Reply:
How many keys would that concern? At a glance I only see acurite, and hideki, 3 keys total. Worth the change.